api/Element.json

```diff
@@ -7005,10 +7005,12 @@
                 "notes": "The <code>block</code> and <code>inline</code> options support the values <code>start</code>, <code>center</code>, <code>end</code>, <code>nearest</code>."
               },
               "safari": {
-                "version_added": false
+                "version_added": "15.4",
+                "notes": "Support the <code>behavior: smooth</code>"
```

Comment:
I think we can ditch this note since we're already talking about it above (and contextually speaking, readers won't be seeing this `options_parameter` feature without seeing its parent)!